layouts/blog/single.html

```diff
@@ -26,7 +26,9 @@ <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl text-center" data-blog-titl
       </h3>
       <div class="flex flex-wrap justify-center items-center my-4">
         <img src="{{ .Params.profile }}" alt="" class="border-4 border-white rounded-full h-11 drop-shadow-lg">
-          <span class="ml-3 blog-date">{{ .Params.author }} | {{ dateFormat "Monday, Jan 2, 2006" .Date }} |&nbsp; </span>
+        {{ range .Params.authors }}
+          <span class="ml-3 blog-date"><a href="/authors/{{ . | urlize }}">{{ . }}</a>  {{ end }} | {{ dateFormat "Monday, Jan 2, 2006" .Date }} |&nbsp; </span>
```

Comment:
The names are not displayed on one line anymore. I'll create a follow on story to fix that.